lib/pengine/unpack.c

```diff
@@ -3826,13 +3831,24 @@ unpack_rsc_op(pe_resource_t *rsc, pe_node_t *node, xmlNode *xml_op,
             goto done;
 
         case PCMK_EXEC_DONE:
+            if (op_is_probe(xml_op) && (rc == PCMK_OCF_NOT_INSTALLED ||
+                                        rc == PCMK_OCF_NOT_CONFIGURED)) {
+                failed_probe = true;
+                break;
```

Comment:
when you rebase this, you'll want this to become a goto done (the failure handling is now after a break)

Reply:
There's no longer any changes in this area, so I don't think I need to do this.